pkg/util/k8sutil/pods.go

```diff
@@ -594,6 +599,9 @@ func CreateArangoSyncPod(kubecli kubernetes.Interface, developmentMode bool, dep
 	}
 	p.Spec.Containers = append(p.Spec.Containers, c)
 
+	// Add priorityClassName
+	p.Spec.PriorityClassName = podPriorityClassName
```

Comment:
Just curious: what is the behavior if you specify a class name of a class which doesn't exist? Do we need to handle something or emit a warning?

Reply:
Pod creation will fail and a event is attached to the arangodeployment resource containing a proper error message from kubernetes.